public/src/js/widgets/fetch-latest-packages.js

```diff
@@ -0,0 +1,37 @@
+import Extension from 'models/extension';
+import {request} from 'modules/authed-ajax';
+import CONST from 'constants/defaults';
+
+export default class extends Extension {
+    constructor(baseModel) {
+        super(baseModel);
+        this.baseModel = baseModel;
+        this.fetchPackages();
```

Comment:
Oh, another thing, extensions can be asynchronous, like this one that's calling the server.

I know that this is not documented anywhere nor easy to find, but in this case extensions should have a [`loaded` ](https://github.com/guardian/story-packages/blob/master/public/src/js/models/extension.js#L7) promise.

`this.loaded = this.fetchPackages();` will do :smile: